SPECS/python-pip/python-pip.spec

```diff
@@ -1,40 +1,37 @@
 %{!?python2_sitelib: %define python2_sitelib %(python2 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
-Name:           python-pip
-Version:        18.0
-Release:        5%{?dist}
-Url:            https://pypi.python.org/pypi/pip
 Summary:        The PyPA recommended tool for installing Python packages.
+Name:           python-pip
+Version:        19.2
+Release:        1%{?dist}
 License:        MIT
-Group:          Development/Languages/Python
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:        https://files.pythonhosted.org/packages/69/81/52b68d0a4de760a2f1979b0931ba7889202f302072cc7a0d614211bc7579/pip-%{version}.tar.gz
+Group:          Development/Languages/Python
+URL:            https://pypi.python.org/pypi/pip
+Source0:        https://files.pythonhosted.org/packages/41/13/b6e68eae78405af6e4e9a93319ae5bb371057786f1590b157341f7542d7d/pip-19.2.tar.gz
```

Comment:
@thcrain-msft suggested a nicer version of the source URL: `https://pypi.io/packages/source/p/pip/pip-%{version}.tar.gz`. This way keeping the `%{version}` macro also makes more sense, since it will keep the URL relevant across version updates.